georaster-layer-for-leaflet.js

```diff
@@ -442,6 +446,56 @@ const GeoRasterLayer = L.GridLayer.extend({
     }
   },
 
+  /**
+   * The callback used to determine the colour based on the values of each pixel
+   *
+   * @callback pixelValuesToColor
+   * @param {array | number} values - The pixel value
+   * @returns {string} - Any valid CSS color string
+   */
+
+  /**
+   * Redraws the active map tiles updating the pixel values using the supplied callback
+   * @param {pixelValuesToColor} pixelValuesToColor - Callback that handles getting the pixel color
+   */
+  updateColors(pixelValuesToColor) {
+    if (!pixelValuesToColor) {
+      console.error("Missing pixelValuesToColor function");
+      return this;
+    }
+
+    if (this.debugLevel >= 1) console.log("Start updating active tile pixel values");
+
+    // update option to ensure correct colours at other zoom levels.
+    this.options.pixelValuesToColorFn = pixelValuesToColor;
+
+    const tiles = this.getActiveTiles();
+    if (!tiles) {
+      console.error("No active tiles available");
+      return this;
+    }
+
+    if (this.debugLevel >= 1) console.log("Active tiles fetched", tiles);
+
+    tiles.forEach(tile => {
+      const { coords, el } = tile;
+      this.drawTile(tile, coords, el.getContext("2d"));
+    });
+    if (this.debugLevel >= 1) console.log("Finished updating active tile colours");
+    return this;
+  },
+
+  getTiles() {
```

Comment:
Thank you.  I think this is a good pattern for highlighting the internal Leaflet Class properties and functions.